[2024-12-18T22:32:08.775+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:32:07.218213+00:00 [queued]>
[2024-12-18T22:32:08.785+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:32:07.218213+00:00 [queued]>
[2024-12-18T22:32:08.786+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-12-18T22:32:08.804+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): env_preparation> on 2024-12-18 22:32:07.218213+00:00
[2024-12-18T22:32:08.811+0000] {standard_task_runner.py:60} INFO - Started process 1635 to run task
[2024-12-18T22:32:08.814+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'env_preparation', 'manual__2024-12-18T22:32:07.218213+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpzrihlosa']
[2024-12-18T22:32:08.816+0000] {standard_task_runner.py:88} INFO - Job 124: Subtask env_preparation
[2024-12-18T22:32:08.868+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:32:07.218213+00:00 [running]> on host 1d3bd1c7da18
[2024-12-18T22:32:08.953+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='env_preparation' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T22:32:07.218213+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T22:32:07.218213+00:00'
[2024-12-18T22:32:08.970+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-18T22:32:08.980+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=env_preparation, execution_date=20241218T223207, start_date=20241218T223208, end_date=20241218T223208
[2024-12-18T22:32:09.027+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-18T22:32:09.055+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
